### DISCUSSION TITLE
Psr\Log version 1?

### DISCUSSION THREAD

**Discussion by xaviermarchegay** [Guikingone/SchedulerBundle#194] [Q&A] [answered]
I would rather open a discussion than a PR.

It is normal that psr\log is still in the version 1? I think that the version 2 could be used (from php 7.4).

Regards

**Comment by Guikingone** [comment, marked as answer]
Hi @xaviermarchegay 👋🏻 

Hum, for now, `psr/log` at `2.0` requires PHP `>=8.0`, this bundle support PHP starting from 7.4 so I think it could be added via:

```json
"psr/log": "^1.0 || ^2.0",
```

WDYT? 

EDIT: I've opened a PR here: https://github.com/Guikingone/SchedulerBundle/pull/195 🙂

**Comment by xaviermarchegay** [comment]
Ah yes,I thought 2.0 was >7.4 and 3.0 was 8.0 but you are right.

3.0 introduces the return types so unless I am mistaken I guess you could technically add 3.0 to the composer.json?

https://github.com/php-fig/log/compare/2.0.0...3.0.0

**Reply by Guikingone** [reply]
Yes, you're right, `3.0` can be added without a lot of troubles, I'm gonna add it and fix the tests, this should be released in `0.7.1` 🙂

**Reply by xaviermarchegay** [reply]
Perfect thanks
